x/project/internal/types/msgs.go

```diff
@@ -80,38 +81,38 @@ func (msg MsgCreateProject) ValidateBasic() sdk.Error {
 	var dataMap ProjectDataMap
 	err := json.Unmarshal(msg.Data, &dataMap)
 	if err != nil {
-		return sdk.ErrInternal(err.Error())
+		return sdkerrors.Wrap(ErrInternal, "not allowed")
 	}
 
 	// Check that evaluatorPayPerClaim is present and is a string integer
 	evaluatorPayPerClaimBz, found := dataMap["evaluatorPayPerClaim"]
 	if !found {
-		return sdk.ErrInternal("missing evaluatorPayPerClaim in project doc")
+		return sdkerrors.Wrap(ErrInternal, "missing evaluatorPayPerClaim in project doc")
 	}
 	var evaluatorPayPerClaimStr string
 	err = json.Unmarshal(evaluatorPayPerClaimBz, &evaluatorPayPerClaimStr)
 	if err != nil {
-		return sdk.ErrInternal(err.Error())
+		return sdkerrors.Wrap(ErrInternal, fmt.Sprintf("%s", err))
```

Comment:
This was not done as I expected it. What I meant was _not_:
```go
return sdkerrors.Wrap(ErrInternal, fmt.Sprintf("%s", err))
```
but just:
```go
return err
```

Reply:
as the err is a type of error..converted to string

Reply:
ok